projects/extension/sql/idempotent/013-loading.sql

```diff
@@ -0,0 +1,83 @@
+-------------------------------------------------------------------------------
+-- loading_row
+create or replace function ai.loading_row
```

Comment:
IMO, `loading_column` would be a more appropriate name.

Reply:
It would also "feel more ergonomic". You're passing a column name to this function

```sql
select ai.loading_column('my_column');
```